File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/OptionsItem.java
package io.github.gusandrianos.foxforreddit.data.models;

import com.google.gson.annotations.SerializedName;

public class OptionsItem{

    @SerializedName("text")
    private String text;

    @SerializedName("id")
    private String id;

    public String getText(){
        return text;
    }

    public String getId(){
        return id;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/OptionsItem.kt
package io.github.gusandrianos.foxforreddit.data.models

import com.google.gson.annotations.SerializedName

class OptionsItem {

    @SerializedName("text")
    val text: String? = null

    @SerializedName("id")
    val id: String? = null

    @SerializedName("vote_count")
    val voteCount: Int? = null
}
